service/src/main/java/greencity/service/HabitServiceImpl.java

```diff
@@ -524,6 +533,45 @@ public void deleteCustomHabit(Long customHabitId, String ownerEmail) {
         habitRepo.save(toDelete);
     }
 
+    /**
+     * Method to like or dislike {@link Habit} specified by id.
```

Comment:
why do you use `HabitVO` in the interface docs and `Habit` here?
maybe it is better to inherit docs from the interface?